Formula/opencascade.rb

```diff
@@ -0,0 +1,41 @@
+class Opencascade < Formula
+  desc "3D modeling and numerical simulation software for CAD/CAM/CAE"
+  homepage "https://www.opencascade.com/content/core-technology"
+  url "http://git.dev.opencascade.org/gitweb/?p=occt.git;a=snapshot;h=refs/tags/V7_2_0;sf=tgz"
+  version "7.2.0"
+  sha256 "adb6fc44efebdb09e30a487fa412afae9599992dc771b0039b4ebf306891a558"
+
+  depends_on "cmake" => :build
+  depends_on "doxygen" => :build
+  depends_on "freeimage"
+  depends_on "freetype"
+  depends_on "gl2ps"
+  depends_on "tbb"
+
+  def install
+    sdk = MacOS::CLT.installed? ? "" : MacOS.sdk_path
+    system "cmake", ".", "-DUSE_FREEIMAGE=ON",
+                         "-DUSE_GL2PS=ON",
+                         "-DUSE_TBB=ON",
+                         "-DINSTALL_DOC_Overview=ON",
+                         "-D3RDPARTY_FREETYPE_DIR=#{Formula["freetype"].opt_prefix}",
+                         "-D3RDPARTY_FREEIMAGE_DIR=#{Formula["freeimage"].opt_prefix}",
+                         "-D3RDPARTY_FREEGL2PS_DIR=#{Formula["gl2ps"].opt_prefix}",
```

Comment:
This should be `"-D3RDPARTY_GL2PS_DIR=#{Formula["gl2ps"].opt_prefix}",`

Reply:
Stepped out for a burger and a beer - will address as some as I get home.  Sorry I botched that!